use-timescale/integrations/query-admin/psql.md

```diff
@@ -1,224 +1,241 @@
 ---
-title: Install the psql connection tool
-excerpt: How to install the psql client for PostgreSQL
+title: Connect to a Timescale Cloud service with psql 
+excerpt: Install the psql client for PostgreSQL and connect to your service 
 products: [cloud, mst, self_hosted]
 keywords: [connect, psql]
 ---
 
-# Install the psql connection tool
+# Connect with psql
 
-The `psql` command line tool is widely used for interacting with a PostgreSQL or
-Timescale instance, and it is available for all operating systems. Most of
-the instructions in the Timescale documentation assume you are using `psql`.
+You use `psql` command line tool to interact with your $SERVICE_LONG. Most procedures in the $COMPANY documentation assume you are using `psql`.
 
-Before you start, check that you don't already have `psql` installed. It is
-sometimes installed by default, depending on your operating system and other
-packages you have installed over time:
+To use `psql` to connect to your database, you need the connection details for your PostgreSQL server. Find those in the connection string generated during service creation. For more information, see [Connecting to Timescale][about-connecting].
```

Comment:
Prerequisites section for consistency.  Use the default one in the partials and add whatever is needed here.